firefox-ios/firefox-ios-tests/Tests/ClientTests/Frontend/Homepage Rebuild/LabelButtonHeaderViewTests.swift

```diff
@@ -0,0 +1,40 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import XCTest
+import Common
+
+@testable import Client
+
+final class LabelButtonHeaderViewTests: XCTestCase {
+    var view: LabelButtonHeaderView!
```

Comment:
Sure I can do this. Is the proposal to have: `let subject = createSubject()` in each test? Do we have a team consensus on unwrapped optionals in tests? [This documentation ](https://github.com/mozilla-mobile/firefox-ios/wiki/Why-and-how-to-write-unit-tests#lifecycle) says:
> The preferred method is to use unwrapped optional

Reply:
Ah ok that is very interesting. I guess that documentation is very old and should be upgraded, since i see that new approach would be to create subject for all tests.

Reply:
I will ask also on the group

Reply:
I think we just merged `force_cast` swiftlint rule, so.... lol

Reply:
I'm happy to disable it in unit tests, personally, as well.